packages/shared/styles/components/organisms/SfHeader.scss

```diff
@@ -1,243 +1,174 @@
-        @include border(--header-navigation-item-border, 0 0 2px 0, solid, transparent);
-        transition: var(--header-navigation-item-transition, border-color 150ms ease-in-out, color 150ms ease-in-out);
-        color: var(--header-navigation-item-color, var(--c-dark-variant));
-        @include font(
-          --header-navigation-item-font,
-          var(--font-weight--normal),
-          var(--font-size--base),
-          1.2,
-          var(--font-family--secondary)
-        );
-        text-transform: uppercase;
-        text-decoration: initial;
-        text-align: center;
-        @include for-desktop {
-          padding: var(--header-navigation-item-padding, calc(var(--spacer-lg) - 1px) var(--spacer-sm));
-          margin: var(--header-navigation-item-margin, 0);
-        }
-        &:focus {
-          --header-navigation-item-color: var(--c-text);
-          --header-navigation-item-border-color: var(--c-primary);
-        }
-      }
-    }
-    &--mobile {
-      --header-navigation-item-padding: 0;
-      --header-navigation-item-border: none;
-      width: var(--header-navigation-item-width, 100%);
-      & > .sf-header-navigation-item__menu-item{
-        --menu-item-font-size: var(--font-size--base);
-        --menu-item-font-weight: var(--font-weight--medium);
-        --menu-item-text-transform: uppercase;
-        display: var(--header-navigation-item-menu-item-display, flex);
-        padding: var(--header-navigation-item-menu-item-padding, var(--spacer-sm));
-        margin: var(--header-navigation-item-menu-item-margin, 0);
-        @include border(--header-navigation-item-menu-item-border, 0 0 1px 0, solid, var(--c-light));
-      }
-    }
-  }
-  
-}
-.sf-header__navigation {
-  display: none;
-  &__menu {
-    display: var(--header-navigation-menu-display, none);
-    @include for-desktop {
-      --header-navigation-menu-display: flex;
-    }
-  }
-  &__sidebar {
-    --sidebar-content-padding: 0;
-    display: var(--header-navigation-sidebar-display, flex);
-    @include for-desktop {
-      --header-navigation-sidebar-display: none;
-    }
-  }
-}
-.sf-header {
-  position: relative;
-  &__wrapper {
-    position: var(--header-wrapper-position);
-    z-index: var(--header-wrapper-z-index, 1);
-    width: 100%;
-    background: var(--header-background, #ffffff);
-    transition: var(--header-wrapper-transition, transform 150ms ease-in-out);
-    transform: var(--header-wrapper-transform);
-    box-shadow: var(--header-box-shadow);
-  }
-  &__header {
-    display: flex;
-    flex-wrap: var(--header-flex-wrap, wrap);
-    align-items: center;
-    justify-content: center;
-    padding: var(--header-padding, 0 var(--spacer-sm));
-    max-width: var(--header-width, 77.5rem);
-    margin: var(--header-margin,0 auto);
-    @include for-desktop {
-      flex-wrap: var(--header-flex-wrap, nowrap);
+  &__logo {    
+    flex: 0 0 auto;
+    margin: var(--spacer-xs) 0;
+    z-index: 1;
+    .sf-image {
```

Comment:
question: I wonder if we shouldnt change this properties via sf-image css variables for height and width?

Reply:
right, I change that.